docs/part3/commonstatsmethods.md

```diff
@@ -1043,14 +1039,81 @@ The output tree will contain the values of the POI that crosses the critical val
 
 You can produce a plot of the value of $p_{x}$ vs the parameter of interest $x$ by adding the option `--plot <plotname>`.
 
-#### Extracting 2D contours
+#### Extracting 2D contours / general intervals
 
-There is a tool for extracting *2D contours* from the output of `HybridNew` located in `test/makeFCcontour.py`. This can be used provided the option `--saveHybridResult` was included when running `HybridNew`. It can be run with the usual <span style="font-variant:small-caps;">Combine</span> output files (or several of them) as input,
+For *two-dimensional* models, or if the parameter does not behave like a cross section, you will need to extract the contours from the output of `HybridNew` and plot them yourself. We will use the `toy-hgg-125.txt` datacard in the example below to demonstrate how this can be done.
```

Comment:
I think it would be good to explicitly include the path to `toy-hgg-125.txt` datacard and to also provide the explicit `text2workspace` command used here (since they have to also include the multiparameter options), just to save people the time of having to figure out these details when they are try to check this example.

Reply:
right, the instructions are already further up the page as its the same as for the multidimfit 2D example, but can repeat the instructions here.

Reply:
True, I think either repeating them, or linking to those instructions would be fine. Though maybe repeating is a bit easier in the end.

Reply:
Agree - done (repeated instructions)